File: day2/tip-calculator.py
print("Welcome to the tip calculator!")
bill = float(input("What was the total bill? $"))
tip = int(input("What percentage tip would you like to give? 10, 12, or 15? "))
split = int(input("How many people will split the bill? "))

bill_with_tip = tip / 100 * bill + bill
bill_per_person = bill_with_tip / split

final_amount = round(bill_per_person, 2)
print(f"Each person should pay ${final_amount})

File: day2/tipCalculator.js
const input = require("readline-sync");

console.log("Hello and welcome to the handy tip calculator");
const bill = parseInt(input.question("How much is the bill? $"));
const tip = parseInt(
  input.question("What percentage would you like to tip? 15, 20, or 25?")
);
const split = parseInt(input.question("How many people are splitting?"));

const billWithTip = (tip / 100) * bill + bill;
let billPerPerson = billWithTip / split;

billPerPerson = billPerPerson.toFixed(2);

console.log(billPerPerson);
